src/marqo/tensor_search/enums.py

```diff
@@ -111,6 +111,10 @@ class EnvVars:
     MARQO_MAX_NUMBER_OF_REPLICAS = "MARQO_MAX_NUMBER_OF_REPLICAS"
     MARQO_BEST_AVAILABLE_DEVICE = "MARQO_BEST_AVAILABLE_DEVICE"
     MARQO_MAX_ADD_DOCS_COUNT = "MARQO_MAX_ADD_DOCS_COUNT"
+    MARQO_OPENSEARCH_MAX_SEARCH_RETRY_ATTEMPTS = "MARQO_OPENSEARCH_MAX_SEARCH_RETRY_ATTEMPTS"
```

Comment:
To be uniform with the other env var names, it may be better to start with `MARQO_MAX`. Also since we're moving to vespa, we could also be backend-agnostic in naming. So something like `MARQO_MAX_BACKEND_RETRY_ATTEMPTS`. Any feedback @pandu-k ?

Reply:
I don't mind too much, the important thing is to clearly communicate what the env var does

Reply:
So can we use `MARQO_MAX_BACKEND_SEARCH_RETRY_ATTEMPTS` and `MARQO_MAX_BACKEND_ADD_DOCS_RETRY_ATTEMPTS`?

Reply:
Wouldn't `MARQO_MAX_BACKEND_RETRY_ATTEMPTS` imply that it will apply to all backend requests as well? I mean we can be specific about it in the documentation but not quite sure if it will be immediately apparent for a user that it only applies to `add_docs()` and `search()` requests for now. Thoughts?

Reply:
@VitusAcabado Yep `MARQO_MAX_BACKEND_RETRY_ATTEMPTS` was a typo, I really meant `MARQO_MAX_BACKEND_SEARCH_RETRY_ATTEMPTS` and `MARQO_MAX_BACKEND_ADD_DOCS_RETRY_ATTEMPTS` to specify that it is for add docs and search.